doc/users/whats_new/extend_kwarg_to_BoundaryNorm.rst

```diff
@@ -0,0 +1,46 @@
+New "extend" keyword to colors.BoundaryNorm
+-------------------------------------------
+
+:func:`~matplotlib.colors.BoundaryNorm` now has an ``extend`` kwarg. This is
+useful when creating a discrete colorbar from a continuous colormap: when
+setting ``extend`` to ``'both'``, ``'min'`` or ``'max'``, the colors are
+interpolated so that the extensions have a different color than the inner
+cells.
+
+Example
+```````
```

Comment:
This example is pretty long for a rarely used kwarg - suggest just linking the real example...

Reply:
Oh hmmm, there is no example.  Suggest this gets put in the appropriate section of `examples/`

Reply:
> This example is pretty long for a rarely used kwarg

I still wonder how people are doing that **without** this kwarg, but well. ;-)

>  Suggest this gets put in the appropriate section of examples/

will do